### DISCUSSION TITLE
Issue Template 변경

### DISCUSSION THREAD

**Discussion by khcho0125** [Team-ComIT/SimTong-Backend#44] [의논하기] [answered]
Issue의 체크 박스가 불필요한 것 같습니다.
Pull request의 체크 박스는 개발을 진행하면서 체크할 수 있지만
Issue의 체크 박스는 주로 Pull request를 마무리한 후 체크하게 됩니다.

**Comment by softpeanut** [comment, marked as answer]
> ### Describe
>
> - <-- 설명 -->
>
> ### Additional
>
> - <-- 추가 설명 -->

그러면 편의를 위해 위와 같은 형식으로 변경할까요?
추가적으로 현재 issue template이 do, fix 2가지 종류가 있는데 리팩터링을 위한 적절한 템플릿이 없는 것 같아 하나 만들고자 하는데 어떤가요?

**Reply by khcho0125** [reply]
좋습니다 👍🏻
